### DISCUSSION TITLE
[Gist] Tab spacing cannot be changed to `4` from `8`

### DISCUSSION THREAD

**Discussion by timothyis** [community/community#38073] [Other Feature Feedback, Questions, & Ideas] [answered]
Hi there,

I'm sorry if this isn't the correct place to provide feedback for gist. I tried to send this through support because It's quite confusing to post feedback to a "community" discussion board. I think it would be incredible to have a "feedback" button along the lines of what Vercel have on every page.

To the issue:
When creating a gist, I accidentally left the indent size to "spaces: 2", but when I changed to "tabs: 4", it automatically sets the tab size to 8. No matter if I change the tab size again to 4, it will remain 8.

Thanks

**Comment by david-engelmann** [comment, marked as answer]
[Check out this answer for help](https://github.com/orgs/community/discussions/23860#discussioncomment-3241924)

**Reply by timothyis** [reply]
Thanks @david-engelmann, I've no idea why that didn't come up in the search. (Probably because the title isn't descriptive 🤦🏼)

However, I still believe this is a bug. The UI of Gist does not make this clear.
